Changes

```diff
@@ -37,6 +37,11 @@ OCaml 4.14 maintenance version
   that causes very slow compaction.
   (Damien Doligez, report by Arseniy Alekseyev, review by Sadiq Jaffer)
 
+- #12513, #12518: Automatically enable emulated `fma` for Visual Studio 2019+
+  to allow configuration with either pre-Haswell/pre-Piledriver CPUs or running
+  in VirtualBox. Restores parity with the other Windows ports, which don't
+  require explicit `--enable-imprecise-c99-float-ops`.
+  (David Allsopp, report by Jonah Beckford and Kate Deplaix, review by ???)
```

Comment:
There is a missing newline, and @shindere ought to be unmasked as a reviewer.

Reply:
Done! 🙂